webv2/web.go

```diff
@@ -435,7 +434,7 @@ func convertSchemaDump(w http.ResponseWriter, r *http.Request) {
 	sessionState.SourceDB = nil
 	sessionState.Dialect = dc.SpannerDetails.Dialect
 	sessionState.SourceDBConnDetails = session.SourceDBConnDetails{
-		Path:           dc.Config.FilePath,
+		Path:           "upload-file/" + dc.Config.FilePath,
```

Comment:
Is this according to the name convention used for all other folders ?

Reply:
will look through code for other naming convention violations and bring them inline as a part of separate exercise since folder creation is not happening as a part of this PR